app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ActiveRecord::Base
+  has_many :follows_followers, class_name: 'Follow', foreign_key: :following_id
+  has_many :follows_following, class_name: 'Follow', foreign_key: :follower_id
+
+  has_many :followers, through: :follows_followers
+  has_many :following, through: :follows_following
+
+  validates_presence_of   :screen_name, :name
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.